jekyll/_cci2/configuration-reference.md

```diff
@@ -27,12 +27,62 @@ You can see a complete `config.yml` in our [full example](#full-example).
 
 Key | Required | Type | Description
 ----|-----------|------|------------
-version | Y | String | `2`, `2.0`, or `2.1` See the Reusing Config doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
+version | Y | String | `2`, `2.0`, or `2.1` See the [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
 {: class="table table-striped"}
 
 The `version` field is intended to be used in order to issue warnings for deprecation or breaking changes.
 
-## **`commands`** (requires version:2.1)
+## **`orbs`** (requires version: 2.1)
+
+Key | Required | Type | Description
+----|-----------|------|------------
+orbs | N | String | The name of an orb to use.
```

Comment:
This table is not correct. The types are not strings. The values of the `commands` map for example, are  themselves commands, as defined below in the `commands` section. @eric-hu can you work on revising this please?

Reply:
Right, I made this up, we need it, but I don't know what the right values are at all for any of these headings, nor what should be in them, maybe this is multiple sections/multiple tables? Thanks @eric-hu for any help you can give me!

Reply:
I think we should consider documenting the use of registered orbs here and link to the inline orbs documentation to talk about the elements of the orbs themselves. That will keep this much cleaner and move discussion of inline orbs to its own area. In that case, we'd say that the value of `orbs` is a map of orb import statements, where the keys are the names used to reference the orb later in config, and the value is an "orb ref" (which we can link to another place or define here -- that will require people understand how orb refs work with versioning, which is likely beyond the scope of this file, so we will probably want to link off to docs on orb refs)

Reply:
```
Key | Required | Type | Description
----|-----------|------|------------
orbs | N | Map | A map of user-selected names to either: orb references (strings) or orb definitions (maps). Orb definitions must be a 2.1 config, though `workflow` and `jobs` are not required.
executors | N | Map | A map of strings to executor definitions. See the Executors Syntax section below.
commands | N | Map | A map of command names to command definitions. See the Commands Syntax section below.
```

@marcomorain @michelle-luna I made a quick pass to update the table. Additionally, the commands section should include something like "a command name cannot be the same as a built-in command name, such as `checkout`".

I also want to call out that I deleted the `jobs` entry from the table. As the markdown table reads now, `orbs`, `exectutors` and `commands` are top-level keys at the same level as `jobs`. That `jobs` part of config was already defined in 2.0 config. (Michelle--let me know if it would be more convenient for me to push changes as a commit to this branch, or if you'd prefer inline comments like this so you can review them as well).

Reply:
thank you @eric-hu, comments are best, so we can all see and discuss them. I've updated the table with your changes.